templates/login.html

```diff
@@ -82,6 +88,14 @@
 
   </div>
 
+  <div class="col-md-8 col-md-offset-2 announcements">
+    <div>
+      <h4>{{ custom.announcement.header }}</h4>
+
+      {{ custom.announcement.text }}
+    </div>
+  </div>
+
```

Comment:
@yuvipanda, I just realized that this is in fact the announcement section. That one is just the message right bellow the login button. See the screenshot in the initial comment

Reply:
Ah, great. Can you make it `announcements` a list that can be iterated instead? And remove the separate option for support link?